### PR TITLE
Docs: Sync the metadata index files, update attributes in doc headers

### PR DESCRIPTION
**Pull request by ebullient**
Quarkus repository half of #28484 

- No blank lines in _attributes*.adoc files
- Enable tests always: toggle between testing just metadata or
  including metadata results with vale linting based on the vale
  system property
- made minor corrections to some document headers.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,7 @@ target
 .project
 .classpath
 .vscode/
+.jbang
```
Comment (maxandersen) [inline .gitignore:10]: this should not need to be ignored. we used it for the revapi tool.

File: core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java [hunk 2]
```diff
@@ -212,7 +212,7 @@ public void afterRestore(Context<? extends Resource> context) throws Exception {
     private static Application manualApp;
     private static final int MANUAL_BEGIN = 0;
     private static final int MANUAL_BEGIN_INITIALIZATION = 1;
-    private static final int MANUAL_INITALIZED = 2;
+    private static final int MANUAL_INITIALIZED = 2;
```
Comment (maxandersen) [inline core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java:215]: what does this change have to do with syncing docs?

File: docs/src/main/asciidoc/_attributes.adoc [hunk 3]
```diff
@@ -1,6 +1,8 @@
+// Common attributes. 
+// --> No blank lines (it ends the document header)
```
Comment (maxandersen) [inline docs/src/main/asciidoc/_attributes.adoc:2]: so these were ignored until now?

Comment (ebullient) [reply docs/src/main/asciidoc/_attributes.adoc:2]: No. things were borked, and nothing cared.

File: core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java (hunk 2 above)
Comment (ebullient) [reply core/runtime/src/main/java/io/quarkus/runtime/Quarkus.java]: I think it was a stray rebase commit. Fixed/removed.

File: docs/sync-web-site.sh [hunk 4]
```diff
@@ -38,12 +38,17 @@ fi
 if [ $BRANCH == "main" ] && [ "$QUARKUS_RELEASE" == "true" ]; then
```
Comment (maxandersen) [inline docs/sync-web-site.sh:38]: running this script fails as it uses git@ based format to clone.

If I change it to https://github.com/quarkusio/quarkusio.github.io it just works.

Comment (maxandersen) [reply docs/sync-web-site.sh:38]: thats probably old issue so not related to this one - worth fixing in separate pr IMO though.

Comment (ebullient) [reply docs/sync-web-site.sh:38]: yes. open a separate issue.

**Review by maxandersen — APPROVED** [review]
+1, looks safe and just add more metadata. LGTM

**Review by gsmet — APPROVED** [review]

**Pull request merged**